handbook.md

```diff
@@ -742,8 +743,8 @@ The goal of Atlantian Youth Rapier is to recreate pre-17th century fencing.  All
         #. Fighters with a valid authorization from another kingdom who move into Atlantia may trade in for an Atlantian authorization card. See the Policies of the MoL for the procedure.
     #. The Kingdom Earl Marshal or Deputy Earl Marshal for Rapier Combat shall warrant a select group of marshals for newly approved authorization forms.  Future authorizations shall follow the standard authorization policies and procedures.
 #. Rapier Combat Weapons and Armor Standards
-    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html),](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
-        #. No spears are allowed.
+    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html)](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
+        #. Only rubber-headed spears are allowed.
         #. Fighters who choose to participate in Cut & Thrust combat acknowledge that they may be struck with a harder blow than they would normally receive in rapier or light rapier combat, and they should act with the interest of the safety of their opponent in mind.
```

Comment:
"may be struck with a harder blow...fighters should protect themselves in armor, clothing and training.   ...should act with the safety of their opponent in mind.  Neither fighter will utilize the rules to gain an unfair or rules lawyer advantage.

Reply:
It is my intent to eventually remove the "may be struck with a harder blow" entirely in a future update.

Reply:
Not arguing.  Is the reasoning the same it was before, or is it an expected change from society?

Reply:
There is no expectation from society relating to this issue.  However, we have seen considerable misunderstanding of this policy, including the mistaken belief that the policy allows for a purposeful increase of force as well as the mistaken belief that a participant's safety is entirely in the hands of their opponents.

I would prefer to simplify this policy as "Fighters are to act with the interest of the safety of themselves and their opponent in mind", but believe this is out of scope of my intent towards bringing our policies inline with society standards.

Reply:
Do we need to even spell out that people need to follow the extant safety rules here?